packages/components/src/components/post-breadcrumb-item/post-breadcrumb-item.scss

```
@@ -0,0 +1,47 @@
+@use 'sass:map';
+@use '@swisspost/design-system-styles/mixins/media';
+@use '@swisspost/design-system-styles/functions/tokens';
+@use '@swisspost/design-system-styles/tokens/components';
+@use '@swisspost/design-system-styles/mixins/utilities';
+
+tokens.$default-map: components.$post-breadcrumb;
+
+:host {
+  display: block;
+}
+
+a {
+  display: inline-flex;
+  align-items: center;
+  padding-block: tokens.get('post-breadcrumb', 'padding-block-text');
```

Can you update the styles to use the tokens function the new way?
```suggestion
  padding-block: tokens.get('breadcrumb-padding-block-text');
```